trezor-wallet-provider.js

```diff
@@ -64,6 +64,41 @@ class Trezor {
     return this.cachedAccounts;
   }
 
+  async signTypedMessage({ from, data }, cb) {
+    from = toChecksumAddress(from);
+
+    const defaultDerivationPath = this.validateFromAndGetDerivationPath(from);
+    const path = this.opts.derivationPath || defaultDerivationPath;
+
+    let response;
+
+    try {
+      await withTempFile(async (file) => {
+        await fs.writeFile(file, data);
```

Comment:
This error is received 

```
The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
```

`data` filed here is of the form object. It can be converted via `JSON.stringify(data)`

Reply:
probably we want to add a type guard first before we stringify it.maybe lodash's `isPlainObject` https://lodash.com/docs/4.17.15#isPlainObject